HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponentsController.swift

```diff
@@ -767,6 +768,26 @@ extension PaymentComponentsController: PaymentReviewProtocol {
     public func openMoreInformationViewController() {
         viewDelegate?.didTapOnMoreInformation()
     }
+
+    /**
+     Fetches bank logos for the available payment providers.
+
+     - Returns: A tuple containing an array of logo data and the count of additional banks, if any.
+     */
+    public func fetchBankLogos() -> (logos: [Data]?, additionalBankCount: Int?) {
+        guard !paymentProviders.isEmpty else { return ([], nil)}
+        let paymentProvidersShownCount = paymentProviders.count == 1 ? 1 : 2
+        let additionalBankCount = paymentProviders.count > 2 ? paymentProviders.count - 2 : nil
+        return (paymentProviders.prefix(paymentProvidersShownCount).map { $0.iconData }, additionalBankCount)
+    }
+
+    private func sortPaymentProviders() {
+        guard !paymentProviders.isEmpty else { return }
+        self.paymentProviders = paymentProviders
+            .filter { $0.gpcSupportedPlatforms.contains(.ios) || $0.openWithSupportedPlatforms.contains(.ios) }
+            .sorted(by: { ($0.index ?? 0 < $1.index ?? 0) })
+            .sorted(by: { ( $0.appSchemeIOS.canOpenURLString() && !$1.appSchemeIOS.canOpenURLString() ) })
```

Comment:
[Suggestion]: please unite those 2 .sorted in one: every sort operation returns  a sorted list which overwrites the previous sort order.